cmd/starlark/starlark.go

```diff
@@ -56,6 +58,16 @@ func doMain() int {
 		}
 		defer pprof.StopCPUProfile()
 	}
+	if *memprofile != "" {
+		f, err := os.Create(*memprofile)
+		if err != nil {
+			log.Fatal(err)
+		}
+		defer func() {
```

Comment:
In the past I used `pprof.WriteHeapProfile`; apparently that's not the new way, though it still works.

We should probably handle the error it returns (even though it is currently sloppy about actually returning errors from the underlying writer).

More importantly, we should also close f, here and for -cpuprofile, and report any error returned by Close. The suggested code in the pprof package doc comments is sloppy about this.

Reply:
Done.